olive/engine/packaging/packaging_generator.py

```diff
@@ -315,24 +311,21 @@ def _download_ort_extensions_package(use_ort_extensions: bool, download_path: st
             run_subprocess(download_command)
 
 
-def _download_c_packages(is_cpu: bool, is_nightly: bool, ort_version: str, download_path: str):
-    NIGHTLY_C_CPU_LINK = Template(
-        "https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly/NuGet/"
-        "Microsoft.ML.OnnxRuntime/overview/$ort_version"
-    )
+def _download_c_packages(is_cpu: bool, ort_version: str, download_path: str):
     STABLE_C_CPU_LINK = Template("https://www.nuget.org/api/v2/package/Microsoft.ML.OnnxRuntime/$ort_version")
-    NIGHTLY_C_GPU_LINK = Template(
-        "https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly/NuGet/"
-        "Microsoft.ML.OnnxRuntime.Gpu/overview/$ort_version"
-    )
     STABLE_C_GPU_LINK = Template("https://www.nuget.org/api/v2/package/Microsoft.ML.OnnxRuntime.Gpu/$ort_version")
     if is_cpu:
-        if is_nightly:
-            urllib.request.urlretrieve(NIGHTLY_C_CPU_LINK.substitute(ort_version=ort_version), download_path)
-        else:
-            urllib.request.urlretrieve(STABLE_C_CPU_LINK.substitute(ort_version=ort_version), download_path)
+        urllib.request.urlretrieve(STABLE_C_CPU_LINK.substitute(ort_version=ort_version), download_path)
     else:
-        if is_nightly:
-            urllib.request.urlretrieve(NIGHTLY_C_GPU_LINK.substitute(ort_version=ort_version), download_path)
-        else:
-            urllib.request.urlretrieve(STABLE_C_GPU_LINK.substitute(ort_version=ort_version), download_path)
+        urllib.request.urlretrieve(STABLE_C_GPU_LINK.substitute(ort_version=ort_version), download_path)
+
+
+def _skip_download_c_package(package_path):
+    warning_msg = (
```

Comment:
From my understanding, even we download the ort package, user still need to install them manually, right? 
Then why not just provide the install command like: pip install ortxxxx==xxx?
I may forget the reason why we need to download the package.

Please correct me if not so.

Reply:
This will save the effort to download packages by themselves. Also, this could be useful for those who want to deploy the output model, so we can provide them a downloaded onnxruntime package.